src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
-    <InputFolder>$(IntermediateOutputPath)input</InputFolder>
-    <OutputFolder>$(IntermediateOutputPath)zoneinfo</OutputFolder>
+    <OutputFolder>$(MSBuildProjectDirectory)/data</OutputFolder>
```

Comment:
Wouldn't this cause to add untracked/staged files to people branch whenever they do a build of this repo? Can we change the way we get package assets for this package to get them from intermediate output?